src/mount/fuse/mount_config.cc

```diff
@@ -114,28 +114,75 @@ void usage(const char *progname) {
 	printf(
 "usage: %s  [HOST[/PORT]:[PATH]] [options] mountpoint\n"
 "\n", progname);
-
-	printf("general options:\n");
 	fuse_cmdline_help();
 
 	printf(
-"SFS options:\n"
+"Aditional info options\n"
+"    -o sfsdebug                 print some debugging information\n"
+"\n"
```

Comment:
Consider moving this one to the end. It is usually for debugging and should not be before the General options.

Reply:
Done